### PR TITLE
feat(frontend): Added support for mysql, mssql and snowflake in the Database Studio

### PR DESCRIPTION
**Pull request by fatonramadani**

### CODE REVIEW

File: frontend/src/lib/components/apps/components/display/dbtable/AppDbExplorer.svelte [hunk 1]
```diff
@@ -69,30 +72,26 @@
...
-	let firstQuicksearch = true
-	$: if (quicksearch) {
-		if (firstQuicksearch) {
-			firstQuicksearch = false
-		} else {
-			aggrid?.clearRows()
-		}
-	}
+	// Everythime quicksearch changes, we want to clear the rows
+	$: quicksearch != undefined && aggrid?.clearRows()
```
Comment (rubenfiszel) [inline frontend/src/lib/components/apps/components/display/dbtable/AppDbExplorer.svelte]: Why was this changed ? Not it clearRows() at initialization which can introduce race issues

Comment (fatonramadani) [reply frontend/src/lib/components/apps/components/display/dbtable/AppDbExplorer.svelte]: Fixed
